packages/utils/plugins-helpers/src/types.ts

```diff
@@ -524,58 +524,60 @@ export namespace Types {
   export type PluginOutput = string | ComplexPluginOutput;
   export type HookFunction = (...args: any[]) => void | Promise<void>;
 
+  export type LifeCycleHookValue = string | HookFunction | (string | HookFunction)[];
+
   /**
    * @description All available lifecycle hooks
    * @additionalProperties false
    */
-  export type LifecycleHooksDefinition<T = string | HookFunction | (string | HookFunction)[]> = {
+  export type LifecycleHooksDefinition = {
```

Comment:
this I guess would be a breaking type change 🤔

Reply:
good point 👍
are those types exported?

should I adjust the changeset?

Reply:
if I read https://unpkg.com/browse/@graphql-codegen/cli@2.14.1/typings/index.d.ts correctly `hooks.ts` is not exported.

So I guess it is not a breaking change - or did I miss sth?

Reply:
Not in Cli but for the `@graphql-codegen/plugin-helpers` which I think many plugins use https://unpkg.com/browse/@graphql-codegen/plugin-helpers@2.7.2/typings/types.d.ts

Reply:
Yes that's absolutely valid 👍 
I believe the risk that a plugin makes use of those generics is very low but it is definitely possible.

Should I change the changeset to be a breaking change?